### PR TITLE
docs: adjust sidebar & create new admin section

### PR DESCRIPTION
**Pull request by yoonhyejin**
## Checklist

- [ ] The PR conforms to DataHub's [Contributing Guideline](https://github.com/datahub-project/datahub/blob/master/docs/CONTRIBUTING.md) (particularly [Commit Message Format](https://github.com/datahub-project/datahub/blob/master/docs/CONTRIBUTING.md#commit-message-format))
- [ ] Links to related issues (if applicable)
- [ ] Tests for the changes have been added/updated (if applicable)
- [ ] Docs related to the changes have been added/updated (if applicable). If a new feature has been added a Usage Guide has been added for the same.
- [ ] For any breaking change/potential downtime/deprecation/big changes an entry has been made in [Updating DataHub](https://github.com/datahub-project/datahub/blob/master/docs/how/updating-datahub.md)

### CODE REVIEW

File: docs/townhalls.md [hunk 1]
```diff
@@ -7,8 +7,9 @@ From time to time we also use the opportunity to showcase upcoming features.
 
 ## Meeting Invite & Agenda
 
-You can join with this link https://zoom.datahubproject.io, or [RSVP](https://rsvp.datahubproject.io/) to get a calendar invite - this will always have the most up-to-date agenda for upcoming sessions.
+You can join with [this link](https://zoom.datahubproject.io) or [RSVP](https://rsvp.datahubproject.io/) to get a calendar invite - this will always have the most up-to-date agenda for upcoming sessions.
 
-## Past Meetings
+## Town Hall History
+
+See our [Town Hall history](https://www.linkedin.com/company/acryl-data/events/) for the recordings and summaries of the past town halls.
```
Comment (hsheth2) [inline docs/townhalls.md:14]: we should say "history after X date is here", history before X date is in this markdown doc

we probably shouldn't delete townhall-history.md outright, but we can hide it from the docs site

**Review by hsheth2 — COMMENTED** [review]
Looks like CI is failing with a broken link

File: docs/townhall-history.md [hunk 2]
```diff
@@ -0,0 +1,527 @@
+# Town Hall History 
+
+:::note 
+For the Town Hall meetings after Mar 2023, please refer to our [LinkedIn Live event history](https://www.linkedin.com/company/acryl-data/events/).
```
Comment (hsheth2) [inline docs/townhall-history.md:4]: This date looks wrong

Comment (yoonhyejin) [reply docs/townhall-history.md:4]: Shoot 🤦‍♀️

Comment (yoonhyejin) [reply docs/townhall-history.md]: Just updated

**Review by hsheth2 — APPROVED** [review]

**Pull request merged**